index.html

```diff
@@ -1,146 +1,19 @@
 <!DOCTYPE html>
-<html>
-
+<html lang="en">
 <head>
-<link rel="stylesheet" type="text/css" href="/css/style.css">
-<script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
-    <title>Property Managemant System</title>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+ <meta http-equiv="X-UA-Compatible" />
+ <title>Document</title>
+<link rel="stylesheet" href="https://use.typekit.net/qhm8wqj.css"/>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css"/>
+<link rel="stylesheet" href="css/style.css"/>
+
 </head>
 <body>
-<header>
-
-<div class="main">
-</div>
-<div class="title">
-<h1>Your Search for Property Begins and Ends here....
-</h1>
-</div>
-<div class="search-box">
-<form action="http://127.0.0.1:8000/front">
-<input class="search-txt" type="text" name="" placeholder="search">
-<a class="search-btn" href="">
-<i class ="fas fa-search"></i>
-
-</a>
-</from>
-</div>
-
-</header>
-<style>
-*{
-margin: 0;
-padding: 0;
-font-family: Century Gothic;
-}
-
-header
-{
-background-image:linear-gradient(rgba(0,0,0,0.5),rgba(0,0,0,0.5)),url(7.jpg);
-height:100vh;
-background-size:cover;
-background-position:center;
-
-}
-.title{
-
-position:absolute;
-text-align:center;
-top:50%;
-left:50%;
-transform:translate(-50%,-50%);
-}
-
-.title h1{
-color:  #fefe10;
-font-size:40px;
-font-weight: bold; 
-}
-ul{
-float:right;
-list-style-type:none;
-margin-top:25px;
-}
-ul li
-{
-display:inline-block;
-}
-ul li a {
-text-decoration:none;
-color:#fff;
-padding:5px 20px;
-border: 1px solid transparent;
-transition:0.6s ease
-}
-ul li a:hover{
-background-color:powderblue;
-color:#000;
-}
-.main{
-max-width: 1200px;
-margin:auto;
-}
-ul li.active a{
-background-color:powderblue;
-color:#000;
-}
-
-.search-box{
-position:absolute;
-top:65%;
-left:50%;
-transform:translate(-50%,-50%);
-background:powderblue;
-height:40px;
-border-radius:40px;
-padding:10px;
-justify-content:center;
-align-items:center;
-}
-.search-box:hover > .search-txt{
-width:240px;
-padding:0 6px;
-}
-
-.search-box:hover > .search-btn{
-background:#000;
-}
-.search-btn{
-color:#e84118;
-float:right;
-width:40px;
-height:40px;
-border-radius:50%;
-background:#2f3640;
-display:flex;
-justify-content:center;
-align-items:center;
-transition:0.4s;
-}
```

Comment:
You have to make changes only the last three sections related to search btn.box...